net/haproxy/Makefile

```diff
@@ -68,18 +68,9 @@ define Package/haproxy-nossl/description
  This package is built without SSL support.
 endef
 
-TARGET=linux-glibc
+TARGET=linux-musl
```

Comment:
Hmmm this doesn't seem to look alright
This is also used by glibc, have you tried to build with it?

Reply:
I think we use MUSL for all targets anyway. I just checked a lot of targets, some from the x86 space, a few from arm and few mips-targets too - they all use MUSL. So, configuring TARGET=linux-musl should be actually correct.

Furthermore, using linux-musl for the TARGET variable does the same thing as linux-glibc except it does not enable the options USE_BACKTRACE and USE_SYSTEMD. Not having those two options is completely fine since we do not care about backtrace(3) or systemd support anyway.

So, all in all, I think using linux-musl is actually perfect and should have been done a long time ago. This change LGTM! Thanks!

Reply:
> I think we use MUSL for all targets anyway. I just checked a lot of targets, some from the x86 space, a few from arm and few mips-targets too - they all use MUSL. So, configuring TARGET=linux-musl should be actually correct.

We use musl by default, but also provided glibc support, users can switch to it manually.

> Furthermore, using linux-musl for the TARGET variable does the same thing as linux-glibc except it does not enable the options USE_BACKTRACE and USE_SYSTEMD. Not having those two options is completely fine since we do not care about backtrace(3) or systemd support anyway.

I disagree using confusing target anyway, and it could lead potential issues in the future.

Reply:
I am sorry to bother you again but I am not sure if I understand correctly. We do not opt for glibc for any target so as a result we use MUSL for _all_ targets. So having TARGET=linux-musl should be perfectly fine, right? So this topic should be resolved or did I miss something?

Reply:
> We do not opt for glibc for any target so as a result we use MUSL for all targets. So having TARGET=linux-musl should be perfectly fine, right

As I said, users can manually switch to glibc (in `Advanced configuration options` -> `Toolchain Options`). It's supported, just not used by default.